Libplanet/Net/Swarm.cs

```diff
@@ -61,14 +62,16 @@ public partial class Swarm : ICollection<Peer>, IDisposable
             IPAddress ipAddress = null,
             int? listenPort = null,
             DateTimeOffset? createdAt = null,
-            IEnumerable<IceServer> iceServers = null)
+            IEnumerable<IceServer> iceServers = null,
+            int protocolVersion = 1)
```

Comment:
I'm not sure if it would be okay to omit `protocolVersion`. but since we need to improve the protocol version system,  I also think it can be addressed at that time.

Reply:
If omitting `protocolVersion` is not proper, I'll change it in a different pull request.